daemon/config_solaris.go

```diff
@@ -0,0 +1,65 @@
+package daemon
+
+import (
+	"github.com/docker/docker/opts"
+	flag "github.com/docker/docker/pkg/mflag"
+	"net"
+)
+
+var (
+	defaultPidFile = "/var/run/docker.pid"
+	defaultGraph   = "/var/lib/docker"
+	defaultExec    = "zones"
+)
+
+// Config defines the configuration of a docker daemon.
+// These are the configuration settings that you pass
+// to the docker daemon when you launch it with say: `docker -d -e lxc`
+type Config struct {
+	CommonConfig
+
+	// Fields below here are platform specific.
+	ExecRoot string `json:"exec-root,omitempty"`
+}
+
+// bridgeConfig stores all the bridge driver specific
+// configuration.
+type bridgeConfig struct {
```

Comment:
Same here, from `bridgeConfig` all this code is duplicated with `config_unix.go`, we might want to have `network_config_unix.go` or `bridge_config_unix.go` to share that :angel: (and thus `config_unix.go`, `config_solaris.go` would be simple :angel: ).